demo.html

```diff
@@ -48,7 +48,8 @@ <h3>and small.</h3>
     <h3>And unordered</h3>
     <div id="cluster3"></div>
     <script src="lib/jquery.js"></script>
-    <script src="lib/d3.v2.js"></script>
+    <script src="https://unpkg.com/d3"></script>
```

Comment:
This stops SVGene from working without internet connection. I'm not really a fan of that.
We could of course think about using a builder like webpack that on build grabs the latest version of d3.